angular/projects/spark-angular/src/lib/components/sprk-modal/sprk-modal.component.ts

```diff
@@ -24,10 +24,12 @@ import * as _ from 'lodash';
       [attr.data-id]="idString"
     >
       <div class="sprk-o-Stack sprk-o-Stack--large">
-        <header class="sprk-o-Stack__item sprk-c-Modal__header">
+        <header class="sprk-c-Modal__header" sprkStackItem>
           <h2
-            class="sprk-c-Modal__heading sprk-b-TypeDisplayFour"
+            class="sprk-c-Modal__heading"
             [id]="heading_id"
+            sprkHeading
```

Comment:
@guern1kn you freaking rock! Thanks for going above and beyond by adding in this sprkHeading directive!!!